### PR TITLE
zh-CN Translation: Translate All Component Introductions Under Reference -> Controls (Excluding Detailed Reference folder)

### PR DESCRIPTION
**Pull request by jindongjie**
- Additionally, explain about **culture-sensitive** under AutoCompleteBox.md for the english version.

- 关于控件名称的翻译我觉得还需要更多人一起抉择，例如 Dock Panel 我翻译成边缘布局面板，但是停靠面板，边缘停靠面板也许是更好的选择。

### CODE REVIEW

**Comment by rabbitism** [comment]
@zdpcdt

**Comment by LoongArum** [comment]
@zdpcdt

**Comment by LoongArum** [comment]
> @zdpcdt

能够先合并吗？已经过了快一个月了，开始有一些分支冲突。

**Comment by rabbitism** [comment]
我粗略看了一下，除了某些属性是否应该翻译存在疑虑，其他都还好，先合并吧，后续有需要再统一更改。

感谢！

**Review by rabbitism — APPROVED** [review]

**Review by Arhell — APPROVED** [review]

**Comment by jindongjie** [comment]
When can we merge this pull request?  :)

**Comment by jindongjie** [comment]
> When can we merge this pull request? :)

@Arhell

File: docs/reference/controls/grid/sharedsizegroup.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
 ﻿---
 id: sharedsizegroup
-title: SharedSizeGroup
```
Comment (maxkatz6) [inline docs/reference/controls/grid/sharedsizegroup.md:3]: This file shouldn't be translated, as it's not in zh-Hans subfolder. Can you revert it?

Comment (jindongjie) [reply docs/reference/controls/grid/sharedsizegroup.md]: Reverted, thanks for catching that!

**Pull request merged**